build/installers/liferay-workspace/liferay-workspace.xml

```diff
@@ -95,7 +95,122 @@
         <addDirectoryToPath>
             <insertAt>end</insertAt>
             <path>${jpmSystemPath}</path>
+            <scope>user</scope>
         </addDirectoryToPath>
+        <if>
+            <explanation>is Mac</explanation>
```

Comment:
no, we don't need to handle this for linux, there is .profile file in $Userhome, add the path will add in it. zsh and csh can read it. For Mac, there is no .profile under userhome, we have to handle mac case.